definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: mixed & {[name: string]: string};
```

Comment:
I think it should actually just be ` {[name: string]: mixed};` I was going to change this myself, thanks for fixing it.